package.json

```diff
@@ -121,10 +121,10 @@
   "dependencies": {
     "axios": "^1.6.8",
     "cors": "2.8.5",
-    "cross-env": "7.0.3",
+    "cross-env": "^7.0.3",
     "dotenv": "^16.4.5",
     "ethers": "^5.2.0",
-    "express": "^4.19.2",
+    "express": "4.19.0",
```

Comment:
Do we have a reason to set version `4.19.0` instead of upgrading to `5.1.0`? `express` is a package that didn't receive many updates in the past years. Recently, they change their policy and they decided to keep updating it. The main changes are for [updating depdencies](https://github.com/expressjs/express/blob/master/History.md)